tests/src/Traits/EntityControllerCacheUtilsTrait.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\Tests\apigee_edge\Traits;
+
+use Drupal\apigee_edge\Entity\Controller\Cache\EntityCacheInterface;
+use Drupal\apigee_edge\Entity\Controller\Cache\EntityIdCacheInterface;
+
+/**
+ * A trait to common functions of entity controller tests.
+ */
+trait EntityControllerCacheUtilsTrait {
+
+  /**
+   * Gets a random unique ID.
+   */
+  protected function getRandomUniqueId(): string {
+    return uniqid('', TRUE);
```

Comment:
You could use the UUID service for this, couldn't you? Or just the `\Drupal\Component\Utility\Random::name()` method.